subprojects/gradle/instrumentation-tests/src/test/kotlin/com/avito/instrumentation/TestRunResultJsonTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.avito.instrumentation
+
+import com.avito.instrumentation.internal.InstrumentationTestsActionFactory.Companion.gson
+import com.avito.instrumentation.internal.TestRunResult
+import com.avito.instrumentation.internal.TestRunResult.Verdict.Failure.Details
+import com.github.salomonbrys.kotson.fromJson
+import com.google.common.truth.Truth
+import org.junit.jupiter.api.Test
+
+class TestRunResultJsonTest {
+    @Test
```

Comment:
seems like detekt not working for test sources, there should be a blank line here

Reply:
Done